ios/Sources/AboutFeature/AboutDroidKaigiFeature.swift

```diff
@@ -0,0 +1,39 @@
+import ComposableArchitecture
+import SwiftUI
```

Comment:
Please do not import `SwiftUI` in Feature file 🙏🏼 

Please implement `UIApplication.shared.open` implementation with divided module.
like this ( `UIApplicationClient` ). https://github.com/DroidKaigi/conference-app-2021/pull/558/commits/d1880e4be62c2588e4ea4a655ca4b27ee7115153

Reply:
OK👍